context/app/static/js/components/detailPage/ProcessedData/ProcessedDataset/DatasetTitle.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import { useHandleCopyClick } from 'js/hooks/useCopyText';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import IconButton from '@mui/material/IconButton';
+import ContentCopyIcon from '@mui/icons-material/ContentCopyRounded';
+import Stack from '@mui/material/Stack';
+import StatusIcon from '../../StatusIcon';
+import { useProcessedDatasetContext } from './ProcessedDatasetContext';
+import VersionSelect from '../../VersionSelect';
+import { useTrackEntityPageEvent } from '../../useTrackEntityPageEvent';
+import SummaryJSONButton from '../../summary/SummaryJSONButton';
```

Comment:
Not for here, but should we add an eslint rule for import order?

Reply:
Yes, I definitely want to get to the bottom of why that's not autofixing right now 😓